.github/workflows/code_ext.yml

```diff
@@ -0,0 +1,82 @@
+name: Code Extention CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - 'master'
+      - 'dev'
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
Since these are 2 different steps, would you also separate them with an empty line?